embassy-usb/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "embassy-usb"
-version = "0.2.0"
+version = "0.2.1"
```

Comment:
If you bumped this only because of my previous review, this should also be removed.